### PR TITLE
Add blog section to landing pages

### PR DESCRIPTION
**Pull request by julianjelfs**
I think this will reasonably easy to maintain. Adding a new post will just be a matter of creating a component with the markup (straight html) and then adding an entry into a dictionary to create a mapping between a slug and the component.

### CODE REVIEW

File: frontend/app/src/components/landingpages/blog/ResponsiveDesign.svelte [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<h1>The Importance of Responsive Web Design for Your Website</h1>
```
Comment (megrogan) [inline frontend/app/src/components/landingpages/blog/ResponsiveDesign.svelte:1]: Did ChatGPT write this?

File: frontend/app/src/components/landingpages/blog/Security.svelte [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<h1>Cybersecurity Best Practices: How to Protect Your Business from Cyber Attacks</h1>
```
Comment (megrogan) [inline frontend/app/src/components/landingpages/blog/Security.svelte:1]: And this?

**Review by megrogan — APPROVED** [review]

File: frontend/app/src/components/landingpages/blog/ResponsiveDesign.svelte (hunk 1 above)
Comment (julianjelfs) [reply frontend/app/src/components/landingpages/blog/ResponsiveDesign.svelte:1]: Yes!

Comment (megrogan) [reply frontend/app/src/components/landingpages/blog/ResponsiveDesign.svelte:1]: Nice!

**Pull request merged**